python/hail/docs/expr/functions/index.rst

```diff
@@ -90,8 +90,8 @@ Functions
     median
     product
     sum
-    unique_max_index
-    unique_min_index
+    argmin
```

Comment:
I like `min_index` and `max_index` better. Poll the group?

Reply:
argmax/argmin are very well-defined terms, but happy to hear more opinions.

Reply:
I see that's what numpy calls it. Leave it as is then.